### ISSUE TITLE
Documentation issue at configuration file section

### ISSUE THREAD

**Issue by tkopra** [MarketSquare/robotframework-tidy#394]
Hi

Noticed issue in https://robotidy.readthedocs.io/en/stable/configuration/config_file.html
```
[tool.robotidy]
src = [
    test.robot
    directory
]
```

Shouldn't that be like this instead?:
```
[tool.robotidy]
src = [
    "test.robot",
    "directory"
]
```

BR,
Tero

**Comment by bhirsz** [comment]
Your'e right, I've actually just fixed it in merged PR - should be fine from the next release.

**Comment by bhirsz** [comment]
Fixed in #370
